src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,49 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  title: string,
+  description: string,
```

Comment:
Great job! Everything is fine! Almost everything) 
You created the same interface so why didn't you use it here?